code/modules/clothing/head/collectable.dm

```diff
@@ -2,8 +2,8 @@
 //Hat Station 13
 
 /obj/item/clothing/head/collectable
-	name = "collectable hat"
-	desc = "A rare collectable hat."
+	name = "collectible hat"
+	desc = "A rare collectible hat."

```

Comment:
'Collectable' means 'a thing that one can collect'. 'Collectible' means
'something of value people would like to have in possession for any
reason'. Of course, just nitpicking here.